docs/UserGuide.adoc

```diff
@@ -562,6 +583,9 @@ that contains the data of your previous ORGANice folder.
 2. Do a tissue typing test to determine the HLA antigens of the patient and the donor.
 3. Do a cross match, which is a trial transfusion.
 
+*Q*: What is the success rate and how is it computed? +
+*A*:
```

Comment:
The success rate measures the tissue compatibility between a donor and a patient. It is computed via this formula: (number of common tissue types between donor and patient) / 6 * 100%

Reply:
Added.